src/pages/documents/pages/users/common/components/Details.tsx

```diff
@@ -12,38 +12,52 @@ import { ValidationBag } from '$app/common/interfaces/validation-bag';
 import { Element } from '$app/components/cards';
 import { Button, InputField } from '$app/components/forms';
 import { cloneDeep, set } from 'lodash';
-import { Dispatch, SetStateAction, useState } from 'react';
+import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import {
-  User,
-  Permission as PermissionType,
-} from '$app/common/interfaces/docuninja/api';
+import { User, Permission as PermissionType } from '$app/common/interfaces/docuninja/api';
 import { DefaultSignature } from './DefaultSignature';
 import { SignatureFontSelector } from '$app/components/SignatureFontSelector';
+import { useUserEditContext } from '../contexts/UserEditContext';
+import { NotificationValue } from '../constants/notifications';
 
+// Legacy props interface for backward compatibility
```

Comment:
I am not sure what backwards compatibility since this is new feature, no? 

Please help me understand.